examples/configuration.json

```diff
@@ -13,5 +13,6 @@
     "output_dim": 1,
     "predicted_value_option": 1,
     "dataset_option": "CuAu_32atoms",
-    "model_type": "PNN"
+    "model_type": "PNN",
+    "out_wunit": "True"
```

Comment:
What about `"normalize_output"` (which would be the opposite)?

Reply:
Fixed in @5948ba39433071174fd06a3ef9b9f90805546870